app/middleware/locals.js

```diff
@@ -1,3 +1,49 @@
+const constants = require('../lib/constants');
+
+function getLocationHeading(query) {
```

Comment:
Can this function be moved into a separate module?
It would facilitate some unit tests (I know there are integration tests already so it might be ok to leave as is) and means this module is just about setting locals.